### ISSUE TITLE
Comments field [COMM] not updating on iTunes/Apple Music

### ISSUE THREAD

**Issue by puyan101** [Marekkon5/onetagger#433]
Hey, love your application and feel like I've finally found the tool that will help me tackle my DJ library and re-structure it.

I've set up the Quick Tag Custom Notes, but I've noticed that anything (Vibe, Mood, etc.) that I assign to the Comments tag frame [COMM] does not update the Comments field on Apple Music.

I plan to use Apple Music's Smart Playlists to organise my music, then export to Rekordbox. However, I noticed that Rekordbox does read the updated values in the [COMM] field when I reload the tag, but Apple Music does not read any updates.

FYI - I currently use MixedInKey before tagging to overwrite the Comments field with Key/Energy. I then plan to add any descriptors after the Key/Energy in the Comments field.

Is this possible or should I amend this somehow to get Apple Music to show the descriptors in the Comments field? Otherwise I will resort to using another field like Grouping/Composer, but this is less preferable.

Thank you! And Happy Holidays.

[onetagger.log](https://github.com/user-attachments/files/18166869/onetagger.log)

**Comment by puyan101** [comment]
Uploading screenshots to show that Rekordbox is able to append the descriptors after the existing Key/Energy from MIK, but Apple Music does not.

![Screenshot 2024-12-17 at 14 32 14](https://github.com/user-attachments/assets/adcb5f3e-78fe-479f-bc8c-031474c1e6a6)
![Screenshot 2024-12-17 at 14 32 55](https://github.com/user-attachments/assets/d667b836-ee08-456f-820b-bcac3c3a38e8)

**Comment by Marekkon5** [comment]
Seems to be similar issue to: https://github.com/Marekkon5/onetagger/issues/317

TL;DR: 
1. Try using latest development build from Actions tab
2. Try setting the ID3 COMM Language to "eng" in Advanced Settings
3. Try playing with ID3v2.3/v2.4 version (in Advanced settings)

**Comment by puyan101** [comment]
Hey, thanks the fast reply and I'm happy to say you helped resolve my problem!
I didn't see #317 when I initially searched, but I tried each step one-by-one and it was the ID3 COMM Language that did the trick. I didn't change the ID3v2.3/v2.4 to even try that, but thanks for helping me out and now I can crack on with tackling my huge library!
Cheers,
P
